docs/sketch-specification.md

```diff
@@ -79,6 +79,14 @@ The `included_libs` key defines the library versions the Arduino Web Editor uses
 Arduino Web Editor specific because all versions of all the Library Manager libraries are pre-installed in Arduino Web
 Editor, while only one version of each library may be installed when using the other Arduino development software.
 
+### Sketch project file
```

Comment:
```suggestion
#### Sketch project file
```

Use H4 heading so that document structure reflects the "sketch project file" is metadata (which is H3 level).

Reply:
Please also add an H4 heading for the `sketch.json` section:

```markdown
#### `sketch.json`
```

Reply:
ok